charts/umbrella/templates/post-install-vault-setup.yaml

```diff
@@ -0,0 +1,43 @@
+{{- if .Values.vault.enabled }}
```

Comment:
could you please still add a file header?

Reply:
done

Reply:
the linter failed unfortunately

Reply:
fixed